mldrivers/linux_bootloaders.ml

```diff
@@ -409,6 +409,28 @@ let detect_bootloader (g : G.guestfs) root i_firmware =
     in
     loop paths in
 
+  (* If we found a grub2 bootloader called /boot/efi/EFI/<OS>/grub.cfg,
```

Comment:
`s/bootloader/boot.cfg/` ?

Reply:
Or "boot config".  Good idea, I'll make this change, thanks.